client/scss/components/_promo.scss

```diff
@@ -162,14 +173,29 @@
   margin-bottom: 0.9em;
   transition: color 400ms ease;
 
-  @include container-query(('l': ('12'))) {
+  @include container-query(('xl': ('12'))) {
     @include font('WB3');
   }
 
   .promo:hover &,
   .promo:focus & {
     color: color('action-green-2');
   }
+
+  .promo--standalone & {
+    @include respond-to('medium') {
+      @include font('WB4');
```

Comment:
Are we going to be doing this a lot with these fonts?
If so could we have a `font-responsive` class?